docs-2.0/3.ngql-guide/4.variable-and-composite-queries/3.property-reference.md

```diff
@@ -82,4 +84,5 @@ nebula> GO FROM "player100" OVER follow YIELD follow._src, follow._dst, follow._
 +-------------+-------------+--------------+--------------+
 ```
 
-The preceding query returns all the neighbors of vertex `"player100"` over edge type `follow`, by referencing `follow._src` as the source vertex ID (which, of course, is `"player100"`) and `follow._dst` as the destination vertex ID.
+The preceding query returns all the neighbors of vertex `"player100"` over edge type `follow`,
+by referencing `follow._src` as the source vertex ID (which is `"player100"`) and `follow._dst` as the destination vertex ID.
```

Comment:
why break the sentence?

Reply:
The mysterious break.

Reply:
It's too long. One line break is the same as a blackspace.